packages/dbos-sqs/index.ts

```diff
@@ -74,7 +75,7 @@ class SQSCommunicator extends ConfiguredInstance
     )
     {
         try {
-            const smsg = {...msg, QueueUrl: msg.QueueUrl || this.config.queueURL};
+            const smsg = {...msg, QueueUrl: msg.QueueUrl || this.config.queueUrl || this.config.queueURL};
```

Comment:
Just so I understand correctly:
- The client can be configured with an  endpoint
- We can also pass a specific queue URL

The AWS SDK will resolve the final destination. But what happens if the client's configured endpoint is different from the queue URL? Which takes precedence?

Reply:
The Queue URL isn't really a URL.  It's got that name for historical reasons.  It is more the queue name.  Though if you haven't given a URL then I guess it's still the URL.  So if you put a valid endpoint and then goofy stuff in the queue URL, it ignores that part of the queue URL and doesn't complain or anything.  (Empirically.  There is no spec for this anywhere I have found.)

Reply:
What it needs to know is what SQS server to contact, who you are, and what the queue is within that.  The service and region basically provide enough to find a server.  The credentials tell it who you are.  The queue name is the only part of the URL it really needs...

Reply:
Got it. We should document what will happen if you use one or the other, or both :)